### PR TITLE
add longitudinal option for cat12vbm and change output name of no bid…

### PR DESCRIPTION
**Pull request by JulieNeuro**
…s quasi raw
To do : 
- Delete brainprep/brainprep/resources/MNI152_T1_1mm_brain.nii.gz, you call the template in spm12 files "/opt..." by default, so no need for this one.
- Change freesurfer version to freesurfer-7.1.1.

### CODE REVIEW

File: brainprep/utils.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def write_matlabbatch(template, nii_file, tpm_file, darteltpm_file, outfile):
     ----------
     template: str
         path to template batch to be completed.
-    nii_files: str
+    nii_files: list
         the Nifti image to be processed.
```
Comment (AGrigis) [inline brainprep/utils.py:108]: image -> images.

File: brainprep/utils.py [hunk 2]
```diff
@@ -114,12 +114,14 @@ def write_matlabbatch(template, nii_file, tpm_file, darteltpm_file, outfile):
         path to the generated matlab batch file that can be used to launch
         CAT12 VBM preprocessing.
     """
-    index = []
-    nii_file = ungzip_file(nii_file, outdir=os.path.dirname(outfile))
+    nii_file_str=""
```
Comment (AGrigis) [inline brainprep/utils.py:117]: not pep8 compliant: missing whitespace around operator.

File: brainprep/utils.py [hunk 3]
```diff
@@ -114,12 +114,14 @@ def write_matlabbatch(template, nii_file, tpm_file, darteltpm_file, outfile):
...
+    for i in nii_file:
+        nii_file_str += "'{0}' \n".format(ungzip_file(i, 
+                                        outdir=os.path.dirname(outfile)))
```
Comment (AGrigis) [inline brainprep/utils.py:120]: not pep8 compliant: continuation line under-indented for visual indent.

File: brainprep/utils.py [hunk 4]
```diff
@@ -114,12 +114,14 @@ def write_matlabbatch(template, nii_file, tpm_file, darteltpm_file, outfile):
...
     with open(template, "r") as of:
         stream = of.read()
-    stream = stream.format(anat_file=nii_file, tpm_file=tpm_file,
-                           darteltpm_file=darteltpm_file)
+    stream = stream.format(anat_file=nii_file_str, tpm_file=tpm_file,
+                        darteltpm_file=darteltpm_file)
```
Comment (AGrigis) [inline brainprep/utils.py:124]: not pep8 compliant: continuation line under-indented for visual indent

File: brainprep/utils.py [hunk 5]
```diff
@@ -114,12 +114,14 @@ def write_matlabbatch(template, nii_file, tpm_file, darteltpm_file, outfile):
         path to the generated matlab batch file that can be used to launch
         CAT12 VBM preprocessing.
     """
-    index = []
-    nii_file = ungzip_file(nii_file, outdir=os.path.dirname(outfile))
+    nii_file_str=""
+    for i in nii_file:
+        nii_file_str += "'{0}' \n".format(ungzip_file(i,
```
Comment (AGrigis) [inline brainprep/utils.py:119]: not pep8 compliant:  trailing whitespace.

**Review by AGrigis — CHANGES_REQUESTED** [review]
PR OK.
The CI is broken on Travis due to the following syntax error (see also the comments in the code):
brainprep/utils.py:117:17: E225 missing whitespace around operator
brainprep/utils.py:119:57: W291 trailing whitespace
brainprep/utils.py:120:41: E128 continuation line under-indented for visual indent
brainprep/utils.py:124:25: E128 continuation line under-indented for visual indent
Thx.

**Comment by JulieNeuro** [comment]
pep8 corrections done

**Pull request merged**